### PR TITLE
Update0418

### PR DESCRIPTION
**Pull request by warmana**
Updates to content, also update to build tooling.

### CODE REVIEW

**Comment by warmanaMOJ** [comment]
Hello @cybersquirrel and @L-Crosby 

Please may I request approval to publish these updates? The changes were requested and have been checked by SPACE team. There are also some minor refreshes to the build tooling to apply security updates.

A full set of the changes introduced in this publication may be seen in the `Files changed` tab at the top of this page.

Many thanks.

**Comment by cybersquirrel** [comment]
Hello @warmanaMOJ  - thanks for sharing this. To check - this is the update to remove all of the Group Security material from the public portfolio?

**Comment by warmanaMOJ** [comment]
Hello @cybersquirrel - yes, exactly that.

File: docs/accessing-moj-it-systems-from-overseas.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This guidance information applies to all staff, contractors and agency staff who
 
 As a government official travelling overseas, you should consider that you may be of interest to hostile parties regardless of your role. By following MoJ policies and processes, you can help reduce the risk to yourself and limit the damage of exposure of sensitive information.
 
-Essential guidance is provided in the [overseas travel information](overseas-travel.md).
+Essential guidance is provided in the overseas travel information available from [MoJ Group Security](mailto:mojgroupsecurity@justice.gov.uk).
```
Comment (cybersquirrel) [inline docs/accessing-moj-it-systems-from-overseas.md:9]: Are Group Security ok with their mailbox being listed externally?

File: docs/accessing-moj-it-systems-from-overseas.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ In general, it is acceptable for MoJ users to access MoJ services from overseas,
 
 -   If you just need to manage your inbox while away, can you delegate permissions to your inbox to a colleague to manage on your behalf?
 
--   Have you left enough time to check and obtain necessary approvals? You should ensure that your request is submitted using the [Overseas Travel Form](/gs/overseas-travel-form.docx) a minimum of fifteen working days before you intend to travel, or if you are subject to STRAP briefing a minimum of twenty working days before you intend to travel. This is because it may be necessary to apply additional technical controls to protect you, your devices, and any data the devices can access.
+-   Have you left enough time to check and obtain necessary approvals? You should ensure that your request is submitted using the Overseas Travel Form available from [MoJ Group Security](mailto:mojgroupsecurity@justice.gov.uk), a minimum of fifteen working days before you intend to travel, or if you are subject to STRAP briefing a minimum of twenty working days before you intend to travel. This is because it may be necessary to apply additional technical controls to protect you, your devices, and any data the devices can access.
```
Comment (cybersquirrel) [inline docs/accessing-moj-it-systems-from-overseas.md:19]: Ditto on this use of the mailbox.

File: dita/cyber-and-technical-security-guidance.dita [hunk 3]
```diff
@@ -161,41 +162,27 @@
       <topic id="prior-to-employment">
         <title>Prior to employment</title>
         <body>
-          <p>These policies are for all users:</p>
+          <p>This policy is for all users:</p>
           <ul>
             <li><xref href="minimum-user-clearance-requirements-guide.dita" format="dita">Minimum User Clearance Levels Guide</xref></li>
```
Comment (cybersquirrel) [inline dita/cyber-and-technical-security-guidance.dita:167]: This seems like an isolated document to retain - is it still needed?

File: docs/security-in-the-office.md
Comment (cybersquirrel) [inline docs/security-in-the-office.md:1]: Should this doc be retained? The content feels duplicative of other docs or largely in the physical security space (e.g. the forwarding of key codes). I'd rather delete this I feel.

**Review by cybersquirrel — COMMENTED** [review]
Thanks for clarification @warmanaMOJ - a few minor comments on the changes (mostly suggesting some additional deletions).

**Comment by warmanaMOJ** [comment]
Hello @cybersquirrel 

Many thanks - very helpful feedback. Will fix all those point.

Thank you again.

**Review by cybersquirrel — APPROVED** [review]
These look good to me - thank you for swift action @warmanaMOJ!

**Pull request merged**